packages/theme/css/characteristics/target.css

```diff
@@ -2,6 +2,8 @@
   --uitk-target-background-hover: var(--uitk-palette-interact-background-hover);
 
   --uitk-target-borderColor-hover: var(--uitk-palette-interact-border-hover);
+  --uitk-target-width-hover: 3px;
```

Comment:
this token needs to be checked with design as other components need to be considered, i assume it isn't needed as a characteristic though as its component specific?

Reply:
@pseys @dplsek